### DISCUSSION TITLE
Manim couldn't find ffmpeg or avconv in juypter notebook

### DISCUSSION THREAD

**Discussion by PompeyLiang** [3b1b/manim#1847] [Q&A] [answered]
Hi guys, I tried to install manim locally for my juypter notebook, but when I finished every set-up and run the code, it shows following error: 
![1658963586(1)](https://user-images.githubusercontent.com/38983015/181388219-aba5fc0b-48be-4959-b37d-62659b11654d.png)

And here is my environment path set-up, I added the ffmpeg to my path already, but still no works:
![1658963700(1)](https://user-images.githubusercontent.com/38983015/181388386-bcc732e0-84bb-4841-8670-f1edb9eaf27e.png)

What is the possible problem?

**Comment by TonyCrane** [comment]
You are using the community version of manim. So please ask this question in their discord channel or open an issue in ManimCommunity/manim instead of here.

**Comment by TonyCrane** [comment, marked as answer]
My suggestion for your problem is to check if ffmpeg.exe is in C:/ffmpeg/bin. If you are sure it is there, maybe you can try restarting your computer.

**Reply by PompeyLiang** [reply]
Thank you Tony, it works now.
